buku

```diff
@@ -618,11 +619,15 @@ class BukuDb:
             return
 
         if not self.json:
-            prompt(results, self.noninteractive)
+            if delete is False:
+                prompt_open(results, self.noninteractive)
```

Comment:
It would be a regular search. No changes there.

Take a look at the keys supported at omniprompt in [googler](https://github.com/jarun/googler#cmdline-options). For example, `f` jumps to the first page, `o` opens the current search in browser.

I am suggesting, for `Buku` we provide a similar key `d` which will delete the search results, *IF* the user presses `d`.